README.md

```diff
@@ -8,21 +8,24 @@ APIs. This allows you to make sure that none of the sub-dependencies try
 anything unexpected, or are able to alter the global objects of your own application
 code, no matter which version you upgrade to.
 
-## Features
+## Security Model
 
 1. Full global isolation: Each top-level dependency is run in a separate container.
 2. Safely restrict the `require` tree to only a subset of allowed modules, including restricting Node.js core access.
```

Comment:
Since it's only two points perhaps just use bullets or sentences?

Reply:
Sure.

Reply:
I did it for:

1. Consistency.
2. We would probably add more.
3. Shit, this is numbered too.